quinn-proto/src/connection/mod.rs

```diff
@@ -3423,6 +3423,11 @@ impl Connection {
         self.state = State::Drained;
         self.endpoint_events.push_back(EndpointEventInner::Drained);
     }
+
+    /// Returns the connection's current MTU
```

Comment:
This just repeats the function's name. Let's add information that helps a reader who doesn't understand the name, and which gives context for how it might be used. Maybe something like:

```
/// Storage required for the largest packet known to be supported by the current path
///
/// Buffers passed to [`Connection::poll_transmit`] should be at least this large.
```